src/Workspaces/Remote/ServiceHub/Services/DesignerAttribute/RemoteDesignerAttributeService.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.DesignerAttribute;
+using Microsoft.CodeAnalysis.SolutionCrawler;
+
+namespace Microsoft.CodeAnalysis.Remote
+{
+    internal partial class RemoteDesignerAttributeService : ServiceBase, IRemoteDesignerAttributeService
+    {
+        public RemoteDesignerAttributeService(
+            Stream stream, IServiceProvider serviceProvider)
+            : base(serviceProvider, stream)
+        {
+            StartService();
+        }
+
+        public Task ScanForDesignerAttributesAsync(CancellationToken cancellation)
```

Comment:
>ScanForDesignerAttributesAsync [](start = 20, length = 30)

Perhaps `StartScanningForAttributesAsync` would be more fitting as this just kicks-off the analyzer

Reply:
done.